skstats/hypotests/calculators/asymptotic_calculator.py

```diff
@@ -1,12 +1,13 @@
-from .basecalculator import BaseCalculator
-from ..fit_utils.utils import eval_pdf, array2dataset, pll
-from ..parameters import POI
 import numpy as np
 from scipy.stats import norm
 
+from .basecalculator import BaseCalculator
+from ..fitutils.utils import eval_pdf, array2dataset, pll
+from ..parameters import POI
+
 
 def generate_asymov_hist(model, params, nbins=100):
-    """ Generate the asymov histogram using a model and dictionnary of parameters.
+    """ Generate the Asymov histogram using a model and dictionary of parameters.
```

Comment:
Sorry for being annoying: Asymov or Asimov?

Reply:
@eduardo-rodrigues I didn't get the first time sorry. Yes indeed it's Asimov my bad.